meta-lmp-base/recipes-support/mfgtool-files/mfgtool-files_0.1.bb

```diff
@@ -9,18 +9,20 @@ inherit deploy nopackages
 INITRAMFS_IMAGE = "fsl-image-mfgtool-initramfs"
 DEPENDS = "${INITRAMFS_IMAGE}"
 
-UUU_RELEASE = "1.5.21"
+UUU_RELEASE = "1.5.109"
```

Comment:
Maybe we could consider using this in the version of recipe name, I mean in the PV.

Reply:
I guess we could break this down into two recipes actually, and incorporate the files and have a proper relationship here, since there are a lot more than just uuu (all the fuse/close/flash scripts).

Reply:
Will merge as it is now as we're reworking this entirely with the new scheme that @angolini is looking at in meta-freescale upstream.